web_ui_next.py

```diff
@@ -8,34 +8,38 @@
 
 SYSTEM_MESSAGE_KEY = "agent_system_message"
 HISTORY_NAME_KEY = "history_name"
+AGENT_NAME_KEY = "agent_name"
 
 
-@st.cache_resource
-def get_agent_store():
-    return AgentStore(
-        "agents",
-        name_to_plugin_class={
-            "PlaywrightPlugin": PlaywrightPlugin,
-        },
-    )
+agent_store = AgentStore(
```

Comment:
The agent store doesn't have any state that needs to be cached. Recreating it everytime allows "warm" reloading of plugins which code has changed.